frontend/app/src/components/home/profile/ReferUsers.svelte

```diff
@@ -92,7 +93,7 @@
                                 size={AvatarSize.Default} />
                         </div>
                         <LinkButton underline="hover">
-                            {$userStore[userId]?.username ?? $_("unknownUser")}
+                            {usernameAndIcon($userStore[userId])}
```

Comment:
this doesn't quite seem to do the same thing. I think if `$userStore[userId]` is undefined you're going to end up with "undefined" rather than the unknown user string.